home/ivp-planning/user-interface/the-3d-panel.md

```diff
@@ -1,28 +1,30 @@
 ---
 description: >-
-  The 3D-Panel acts primarily as visualization area. Like in the 2D-panel it
-  where you can place, move, scale and rotate machines and objects as if you
+  The 3D-Panel acts primarily as visualization area. Like in the 2D-panel you can place, move, scale and rotate machines and objects as if you
   would work with a traditional floor plan on paper.
 ---
 
 # 3D-Panel
 
-The 3D-Panel acts as a visualization area where you can work in a three dimensional representation of the room or factory. It is possible to [place](../machines/first-steps-with-3d-object.md), [move](../machines/selecting-and-moving-objects.md), [scale and rotate](../machines/scale-and-rotate-objects.md) machines and objects like in the 2D-panel. However, the 3D-panel also offers functions to take [screenshots ](../advanced-tools/creating-screenshots.md)or to work with [virtual cameras](virtual-camera-panel.md) to [record videos](../advanced-tools/virtual-cameras.md).
+The 3D-Panel acts as a visualization area where you can work in a three dimensional representation of the room or factory. It is possible to [place](../machines/first-steps-with-3d-object.md), [move](../machines/selecting-and-moving-objects.md), [scale](../machines/scale-objects.md) and [rotate](../machines/rotate-objects.md) machines and objects like in the 2D-panel. However, the 3D-panel also offers functions to take [screenshots ](../advanced-tools/creating-screenshots.md)or to work with [virtual cameras](virtual-camera-panel.md) to [record videos](../advanced-tools/virtual-cameras.md).
```

Comment:
rotate-objects.md does not exist in this branch. Should this change have been in https://github.com/iXtenda/iVP-gitbook/pull/28?